dts/bindings/mtd/jedec,spi-nor-common.yaml

```diff
@@ -8,13 +8,22 @@
 properties:
   jedec-id:
     type: uint8-array
-    required: true
+    required: false
```

Comment:
I assume the change here and to (`size`) is to make these optional if `sfdp-bfp` is defined?  If so can we maybe update the descriptions of both to say that (ie required if `sfdp-bfp` is not defined, ...)